modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_card_deck.yml

```
@@ -0,0 +1,79 @@
+id: az_paragraph_card_deck
+label: AZ Paragraph Card Deck
+migration_group: az_migration
+migration_tags:
+  - Quickstart Content Migration
+status: true
+
+source:
+  plugin: az_paragraphs_item
+  bundle: uaqs_card_deck
+  field_collection_names: 'field_uaqs_cards'
+
+destination:
+  plugin: 'entity_reference_revisions:paragraph'
+  default_bundle: az_cards
+
+process:
+
+  field_az_cards:
+    -
+      plugin: sub_process
+      source: field_uaqs_photo
+      process:
+        delta: delta
+        media: 
+          - 
+            plugin: migration_lookup
+            source: fid
+            migration: 
+              - az_media
+    -
+      plugin: paragraphs_field_property_mapping
+      key: title
+      field_name: field_uaqs_short_title
+      value: value
+    -
+      plugin: paragraphs_field_property_mapping
+      key: body
+      field_name: field_uaqs_summary
+      value: value
+    -
+      plugin: paragraphs_field_property_mapping
+      key: body_format
+      field_name: field_uaqs_summary
+      value: az_standard
+    -
+      plugin: paragraphs_field_property_mapping
+      key: link_title
+      field_name: field_uaqs_links
+      value: title
+    -
+      plugin: paragraphs_field_property_mapping
+      key: link_uri
+      field_name: field_uaqs_links
+      value: url
+    -
+      plugin: paragraphs_field_property_mapping
+      key: options
+      value: 
+        class: 'bg-white'
+
+  behavior_settings:
+    plugin: paragraphs_behavior_settings
+    card_width: 'col-md-4 col-lg-4'
+    card_style: 'card card-borderless'
+    card_width_sm: 'col-sm-6'
+    card_width_xs: 'col-12'
+    card_width_field: field_uaqs_setting_deck_width
+    card_count: field_uaqs_photo
```

I think it would be good to name the `card_count` configuration property `card_count_field` for consistency.
```suggestion
    card_count_field: field_uaqs_photo
```